disttask/framework/dispatcher/dispatcher_test.go

```diff
@@ -145,7 +145,7 @@ const (
 )
 
 func checkDispatch(t *testing.T, taskCnt int, isSucc bool, isCancel bool) {
-	failpoint.Enable("github.com/pingcap/tidb/domain/MockDisableDistTask", "return(true)")
+	require.NoError(t, failpoint.Enable("github.com/pingcap/tidb/domain/MockDisableDistTask", "return(true)"))
```

Comment:
Why here need call require.NoError? for testing stable?

Reply:
> Why here need call require.NoError? for testing stable?

It's better to handle all errors.